detection-rules/link_login_or_captcha.yml

```diff
@@ -6,15 +6,13 @@ severity: "high"
 source: |
   type.inbound 
   and any(body.links, 
-      all([beta.linkanalysis(.)], (
-          .credphish.contains_login
-          or .credphish.contains_captcha
+      beta.linkanalysis(.).credphish.contains_login
+          or beta.linkanalysis(.).credphish.contains_captcha
+      )
+      and any(body.links,
+          beta.linkanalysis(.).effective_url.domain.root_domain in $free_subdomain_hosts
+          or .href_url.domain.root_domain in $free_subdomain_hosts
```

Comment:
```suggestion
      beta.linkanalysis(.).credphish.contains_login
      or beta.linkanalysis(.).credphish.contains_captcha
  )
  and any(body.links,
          beta.linkanalysis(.).effective_url.domain.root_domain in $free_subdomain_hosts
          or .href_url.domain.root_domain in $free_subdomain_hosts
  )
```

Reply:
Actually the additional paren inclusion now throws a syntax highlight..